packages/amplication-plugin-api/src/pluginVersion/base/PluginVersion.ts

```diff
@@ -25,6 +25,17 @@ class PluginVersion {
   @Field(() => Date)
   createdAt!: Date;
 
+  @ApiProperty({
+    required: false,
+    type: String,
+  })
+  @IsString()
+  @IsOptional()
+  @Field(() => String, {
+    nullable: true,
+  })
+  deprecated!: string | null;
+
```

Comment:
Did you add it manually or with Amplication? (same for the Prisma schema file change)?

Reply:
I've manually cherry picked the amplication branch changes related to the deprecated field.